project/Build.scala

```diff
@@ -775,14 +775,14 @@ object Build {
   lazy val dottyLibrarySettings = Seq(
     libraryDependencies += "org.scala-lang" % "scala-library" % scalacVersion,
     // Add version-specific source directories:
-    // - files in src-scala3 will only be compiled by dotty
-    // - files in src-scala2 will only be compiled by scalac
+    // - files in src-bootstrapped will only be compiled by dotty-bootstrapped
```

Comment:
It's actually compiled by the non-bootstrapped compiler

Reply:
or more correctly, it's compiled by both the non-bootstrapped and bootstrapped compiler.